wordpress-mobile/WordPress-Android
# Feature/quick start task counter

Part of #7707 

This PR adds:

- Quick Start task counter
- All task completion indicator (orange dot)
- Functionality to hide the quick start after all task have been skipped OR all tasks have been completed and the list was checked one last time.
- Improved design of the Quick Start fragment.

To test:

1. Complete quick start tasks and make sure the counter updates.
2. When all the tasks are completed, make sure the orange dot is displayed near the task counter.
3. After checking Quick Start list when an orange dot is visible make sure it disappears from the MySite options list.
4. Skipping all tasks from Quick Start list should immediately hide Quick Start row from MySite fragment (@SylvesterWilmott could you confirm that this is intended behavior?)

PS: You can reset quick start flag with `AppPrefs.setQuickStartActive(true);`

Orange dot of doom
![screenshot-2018-07-25_23 53 31 796](https://user-images.githubusercontent.com/728822/43229710-06a57ea4-9066-11e8-8f9d-cca53cb53142.png)

Updated Quick Start task likst design:
![screenshot-2018-07-25_23 57 19 73](https://user-images.githubusercontent.com/728822/43229861-8258b37c-9066-11e8-87d0-f6151a6031dd.png)
